docs/source/tutorial/registration.md

```diff
@@ -57,9 +57,13 @@ types of losses:
 ### Intensity based (image based) loss
 
 This type of loss measures the dissimilarity of the fixed image and warped moving image,
-which is adapted from the classical image registration methods. Intensity based loss is
-modality-independent and similar to many other well-studied computer vision and medical
-imaging tasks, such as image segmentation.
+which is an adaptation from classical image registration methods. Intensity based losses
+can perform poorly on multi-modality registrations (for example, SSD loss in CT-MRI
+registration). Generally, intensity based losses work best when there is an inherent
+consistency in appearance between moving and fixed images, which is more common in
+single-modality registration. Intensity based losses can be used with any modality and
```

Comment:
Good to make it clearer and specific. But as is would make people claiming mutual information works between different MR sequences (modalities) not very happy :) suggest to add: ... although some loss formulation such as mutual information may be effective between specific pairs of image modalities, such as some MR sequences. (or something like that)

Reply:
Changed paragrah to: "Intensity based losses measure the dissimilarity between a fixed image and a warped
moving image, which is an adaptation from classical image registration methods. These
losses can perform poorly on multi-modality registrations (e.g. SSD loss in CT-MRI
registration), although certain multi-modality tasks, such as registration between
different MRI sequences are handled well by MI. Generally, intensity based losses work
best when there is an inherent consistency in appearance between moving and fixed
images, which is more common in single-modality registration."